server/fleet/software.go

```diff
@@ -2,9 +2,13 @@ package fleet
 
 import "time"
 
-type SoftwareCVE struct {
+type CVE struct {
 	CVE         string `json:"cve" db:"cve"`
-	DetailsLink string `json:"details_link" db:"details_link"`
+	DetailsLink string `json:"details_link" db:"-"`
+	// These are double pointers so that we can omit them AND return nulls when needed.
```

Comment:
Nice trick! While we're at it, let's clarify why we need to support "omitted" and `null`. My guess for the three "types of values":
1. omitted for non-premium installs.
2. `null` for premium but doesn't have a value set for the field.
3. value set for premium.

Reply:
fixed in 67d0e1ca9

Reply:
Is the comment gone in the last commit for some reason?

Reply:
not sure how that happened, fixed